### DISCUSSION TITLE
How to set allow_mutation = False

### DISCUSSION THREAD

**Discussion by jinserk** [collerek/ormar#238] [Q&A] [answered]
Hi,

Another question for pydantic Config.
Once a obj created and saved, I want to make it immutable. In `pydantic` model, I could just give `allow_mutation = False` in `Config`. How can I do this in `ormar` model?

Thank you!

**Comment by collerek** [comment, marked as answer]
As of now Config is generated for you (and will be overwritten even if you provide yours).
Used config is:
```
class Config(pydantic.BaseConfig):
    orm_mode = True
    validate_assignment = True
```

Config internally is stored in a model class under `__config__` key so you can alter the settings this way:

```python
class Mol(ormar.Model):
    ... # class declaration

Mol.__config__.allow_mutation = False
```

**Reply by jinserk** [reply]
That's what I want to know! Thank you so much! :)
